packages/j/juliamono.json

```
@@ -14,14 +14,16 @@
   },
   "license": "OFL-1.1",
   "homepage": "https://github.com/cormullion/juliamono",
+  "filename": "webfonts/juliamono-regular.css",
```

Should it maybe be this, because `basePath = "webfonts"`?

```suggestion
  "filename": "juliamono-regular.css",
```

---

That might be a better guess than mine! :)